src/Configuration.cs

```diff
@@ -24,15 +24,18 @@ public class Configuration : IPluginConfiguration
         public bool WheelEnabled = true;
         [JsonIgnore]
         public bool WindowMoveableGW = false;
-        public bool ShowOutOfCombatGW = false;
-        public bool ShowOnlyGCDRunningGW = false;
-        public bool WheelQueueLockEnabled = true;
+        public bool ShowOutOfCombat = false;
+        public bool HideAlertsOutOfCombat = false;
+        public bool HideIfTP = false;
```

Comment:
Sorry, which one in particular?

Reply:
Hide If TP. It feels like everyone that has Show on GCD Rolling would want this turned on.

Reply:
> Hide If TP. It feels like everyone that has Show on GCD Rolling would want this turned on.

There are a couple of other Hides that I'll get around to implementing soon-ish.  Items for one.  I also need to figure out how to distinguish when a player is interacting with a clickable "cast" like attuning to an Aetheryte.  That's gonna be a bit of a chore.

Reply:
Just a heads up before you delve into that. I looked at it some time ago, and I believe you can get that information from calling `GetRecastGroupDetail`, `GetAdditionalRecastGroup` or some other similar function on the Dalamud interface. There was a group that handled TPs and another for items. Not sure if that would also work for clickable casts but if it works it may make handling this easier.